pkg/codegen/testing/test/testdata/output-funcs/nodejs/funcWithEmptyOutputs.ts

```diff
@@ -7,7 +7,7 @@ import * as utilities from "./utilities";
 /**
  * n/a
  */
-export function funcWithEmptyOutputs(args: FuncWithEmptyOutputsArgs, opts?: pulumi.InvokeOptions): Promise<FuncWithEmptyOutputsResult> {
+export function funcWithEmptyOutputs(args: FuncWithEmptyOutputsArgs, opts?: pulumi.InvokeOptions): Promise<void> {
```

Comment:
This is a breaking change. While I think the resulting code is better, I would do this in a separate PR so it is independently reviewable and revertible.